File: Module 3/Mastering Swift 2_Code/B05115_10_codes/TestObjectiveC/TestObjectiveC/ViewController.m
//
//  ViewController.m
//  TestObjectiveC
//
//  Created by Jon Hoffman on 1/4/15.
//  Copyright (c) 2015 Jon Hoffman. All rights reserved.
//

#import "ViewController.h"
#import "MyObject.h"

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];

    // Do any additional setup after loading the view.
    
    MyObject *m;
 //   m = [[MyObject alloc]init];
    NSLog(@"Value: %d",[m myMethodWithValue:5]);
    
}

- (void)setRepresentedObject:(id)representedObject {
    [super setRepresentedObject:representedObject];

    // Update the view, if already loaded.
}

@end

File: Module 3/Mastering Swift 2_Code/B05115_08_codes/Samples/Swift_XML_OSX/Swift_XML_OSX/ViewController.swift
//
//  ViewController.swift
//  Swift_XML_OSX
//
//  Created by Jon Hoffman on 12/27/14.
//  Copyright (c) 2014 Jon Hoffman. All rights reserved.
//

import Cocoa

class ViewController: NSViewController {
    
    var xmlString = "<?xml version=\"1.0\"?><books><book name=\"iOS and OS X Network Development Cookbook\"><author>Jon Hoffman</author><publisher>PacktPub</publisher><category>Programming</category><description>Network development for iOS and OS X</description></book><book name=\"Mastering Swift\"><author>Jon Hoffman</author><publisher>PacktPub</publisher><category>Programming</category><description>Learning Swift</description></book></books>"

    override func viewDidLoad() {
        super.viewDidLoad()

        // Do any additional setup after loading the view.
        
        let xmlParser = MyXMLParser()
        xmlParser.parseXmlString(xmlString)

    }

    override var representedObject: AnyObject? {
        didSet {
        // Update the view, if already loaded.
        }
    }


}

